README.md

```diff
@@ -2,5 +2,16 @@
 
 Language server for validating and providing hints for writing Cloudify blueprints.
 
-Install cfy-lint in path for linting in vscode.
+
+## Requirements:
+
+ - Python 3.6, or higher.
+ - Install cfy-lint in path for linting in vscode.
+
+ ## How to install cfy-lint
+ ```bash
+   git clone https://github.com/cloudify-cosmo/cfy-lint.git
+   pip install -e .\cfy-lint\
```

Comment:
Let's replace the git clone and the `pip install -e` and replace it all with just `pip install cfy-lint`.